.github/workflows/update-docs-webhook.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
     steps:
       - name: Call deployment webhook
         env:
-          WEBHOOK_URL: ${{ secrets[AMPLIFY_DOCS_DEPLOY_HOOK] }}
+          WEBHOOK_URL: ${{ secrets["AMPLIFY_DOCS_DEPLOY_HOOK"] }}
```

Comment:
nit: Why use this syntax instead of the `secrets.AMPLIFY_DOCS_DEPLOY_HOOK` used everywhere else?

Reply:
It was done this way to support matrix, when we had to call two webhooks:
- https://github.com/gravitational/teleport/blob/b84f071fe07bfc0579f1dcf1a05efa502a2be45d/.github/workflows/update-docs-webhook.yaml#L18-L31

Now when second webhook and matrix is removed, there is no real reason to keep it this way.

Reply:
Updated in https://github.com/gravitational/teleport/pull/50778/commits/a9b77651fdcc3d46adb7de4943368b85c4ca08d9